src/main/java/com/github/martapietka/ppm2pbm/PpmToPbmConverter.java

```diff
@@ -24,5 +23,23 @@ public static void convert(InputStream inputStream, OutputStream outputStream) t
         while (inputStream.read() > 0x1F) {
             // skip comment
         }
+
+        byte[] newHeader = {header[0], 0x31, header[2]};
+
+        int width = ByteReader.convertBytesToInt(inputStream);
+        int height = ByteReader.convertBytesToInt(inputStream);
+
+        String newWidth = Integer.toString(width);
```

Comment:
To nie jest _nowa_ wysokość, po prostu inna reprezentacja tej, co była 😉 proponuję `widthString` i analogicznie z szerokością.